src/content/tools/pub/cmd/pub-unpack.md

```diff
@@ -0,0 +1,89 @@
+---
+title: dart pub unpack
+description: Downloads a package and unpacks its contents in place.
+---
+
+:::version-note
+The `unpack` subcommand was introduced in Dart 3.4.
+To download the archive of a package with an earlier SDK,
+visit the **Versions** tab of a package on the pub.dev site.
+:::
+
+_Unpack_ is one of the commands of the [pub tool](/tools/pub/cmd).
+
+```plaintext
+$ dart pub unpack <package>[:descriptor] [--[no-]resolve] [--output=<output directory>] [--[no-]force] [other options]
+```
+
+This command downloads the specified `<package>`
+according to the specified descriptor then
+extracts its contents to a `<package>-<version>` directory.
+
+For example, the following command downloads and extracts the
+latest stable version of `package:http` from the [pub.dev site]({{site.pub}}),
+to the current directory:
+
+```console
+$ dart pub unpack http
+```
+
+By default, pub runs `dart pub get` after the package is extracted
+to resolve the package's dependencies.
+To disable the automatic resolution, pass the `--no-resolve` option:
+
+```console
+$ dart pub unpack http --no-resolve
+```
```

Comment:
Since this is covered word for word in the options sections, I'd remove it from here
```suggestion
```